config/configure.in

```
@@ -563,6 +563,40 @@ else
   AC_MSG_RESULT([found $NUMS_INCLUDE])
 fi
 
+CURL=yes
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking ocaml curl library" >&5
+AC_MSG_CHECKING([ocaml curl library]); }
+CURL_INCLUDE=$(ocamlfind query -i-format curl 2> /dev/null)
+if test $? -ne 0; then
+  CURL=no
+fi
+CURL_LIB=$(ocamlfind query -a-format -predicates native curl 2> /dev/null)
+if [ $? -ne 0 ]; then
+  CURL=no
+fi
+CURL_CLIB=$(ocamlfind query -l-format -predicates native curl 2> /dev/null)
+if [ $? -ne 0 ]; then
```

You missed converting these brackets too.

```suggestion
if test $? -ne 0; then
```